src/components/select/test/select.test-wrapper.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @stencil/required-jsdoc */
```

Comment:
We should perhaps just disable this rule for all test-files?

Reply:
Yeah that should be fine :slightly_smiling_face: